bot.py

```diff
@@ -134,9 +135,27 @@ async def on_message(self, message):
         # Execute the command.
         await handler(message, command_content)
 
-    def next_in_queue(self, _):
+    def _stop(self):
+        """
+        voice_client.stop() triggers the after callback, so we need to set a state so it knows
+        not to go to the next item in queue.
+        """
+        self.block_after = True
+        self.voice_client.stop()
+
+    def after_callback(self, _):
+        """
+        Goes to the next item in queue if the callback was triggered after a media finished
+        playing normally (i.e. not from voice_client.stop())
```

Comment:
Nit: Someone could call voice_client.stop() with block_after == False so the comment isn't entirely accurate.

```
Plays the next item in queue if block_after == False, otherwise stops the music. Used as a callback for play().
```

Reply:
yes

Reply:
I agree with this. I have changed the comments based on your suggestion.